nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -164,4 +165,32 @@ private void appendToFile(String row, Path path) {
       throw new RuntimeException("Error happened for writing at file: " + path, e);
     }
   }
+
+  /**
+   * Converts the given uri to the real path. Note, in NullAway CI tests, source files exists in
+   * memory and there is no real path leading to those files. Instead, we just serialize the path
+   * from uri as the full paths are not checked in tests.
+   *
+   * @param uri Given uri.
+   * @return Real path for the give uri.
+   */
+  @Nullable
+  public static Path pathToSourceFileFromURI(@Nullable URI uri) {
+    if (uri == null) {
+      return null;
+    }
+    if ("jimfs".equals(uri.getScheme())) {
```

Comment:
Add a comment indicating this case is for NullAway unit tests?

Reply:
I added a comment for that. aaf73e54265008c96de36addfc715bb7d2178322